packages/sommelier.rb

```diff
@@ -3,7 +3,7 @@
 class Sommelier < Package
```

Comment:
Please add this change as a separate PR.  We need to keep these things separate because we don't have the binaries yet.  With that said, you will also need to remove the pre-built binaries.

Reply:
No, I won't. I just changed the postinstall.

Reply:
Just like with gtk_doc. No difference in the files.

Reply:
Actually that is correct but I still want a separate PR. :)

Reply:
I will revert sommelier in this one, and after this is merged I will create a new one.